boxoffice/__init__.py

```diff
@@ -1,76 +1,35 @@
 # -*- coding: utf-8 -*-
 # imports in this file are order-sensitive
-import re
 from pytz import timezone
 from flask import Flask
-from flask.ext.assets import Environment, Bundle
+from flask.ext.mail import Mail
 from flask.ext.lastuser import Lastuser
 from flask.ext.lastuser.sqlalchemy import UserManager
-from baseframe import baseframe, baseframe_js, baseframe_css
 import coaster.app
+import wtforms_json
 
 app = Flask(__name__, instance_relative_config=True)
 lastuser = Lastuser()
+ALLOWED_ORIGINS = ['http://shreyas-wlan.dev:8000',
+                   'http://rootconf.vidya.dev:8090',
+                   'http://rootconf.karthik.dev:8090']
 
-assets = Environment(app)
-js = Bundle(baseframe_js,
-    filters='jsmin', output='js/packed.js')
-css = Bundle(baseframe_css, 'css/app.css',
-    filters='cssmin', output='css/packed.css')
-assets.register('js_all', js)
-assets.register('css_all', css)
+mail = Mail()
 
 
 # Configure the app
 def init_for(env):
     coaster.app.init_app(app, env)
+    db.init_app(app)
+    db.app = app
+
     lastuser.init_app(app)
-    lastuser.init_usermanager(UserManager(boxoffice.models.db, boxoffice.models.User))
+    lastuser.init_usermanager(UserManager(boxoffice.models.db,
+                                          boxoffice.models.User))
```

Comment:
As per hgapp conventions, you import both `models` and `db` in this file, so you don't need the full dotted name here.